modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ImagePicker/ImagePicker.es.js

```diff
@@ -99,7 +105,10 @@ const ImagePicker = ({
 				Liferay.Language.get('select-x'),
 				Liferay.Language.get('image')
 			),
-			url: itemSelectorURL,
+			url: addParams(
+				`selectedFileEntryId=${selectedImageId}`,
```

Comment:
Shouldn't we use a more generic name (e.g. `selectedEntryId`)? I think we'll need to apply similar changes to other item selectors that support folders (KB, Journal) where that param name doesn't make sense.

Reply:
I see.... what about `selectedItemId`? Anyway I still have doubts, what if we are opening a multiple selection Item Selector? Should we pass always an array to cover all the cases?

Reply:
`selectedItemIds` is fine too. Passing a single element is a special case of passing multiple ones, so its probably better to pass the array.